resources/nix/README.md

```diff
@@ -0,0 +1,21 @@
+# Nix Packages
+
+[Nix] is a package manager for Linux and macOS. It targets reproducible builds by providing fixed versions of packages and their dependencies.
+
+## Derivations
+
+Nix itself does not hold any executables. Instead, it offers a script that tells Nix how to acquire the executable by, for example, downloading a release version from GitHub. This script is called a derivation. Derivations are written in the [Nix functional language].
+
+The [Nix package manager] holds a collection of many derivations for popular software. This collection can be extended by providing so-called [Overlays]. Overlays are derivations that extend the collection of derivations provided by the package manager. Overlays are usually used when the users cannot publish their derivations in the package manager's collection.
```

Comment:
Not yet, but especially for the third parties, that might be interesting. Overlays would allow getting rid of them and therefore I thought it might be interesting to add this information for future PRs.

Reply:
I doubt anybody is going to do that in the near future. For that time being, it is confusing information.
Can you please remove it?

What might  make sense would be something like `// TODO(anybody): consider using overlays for third party libraries'. But since this is not a functional problem but rather a "nice to have", I would even leave out the todo. Honestly, nobody is going to do it. There are more pressing topics.

Reply:
I see, you are right. I removed it 👍